desc/objectives/objective_funs.py

```diff
@@ -130,12 +170,53 @@ def build(self, use_jit=None, verbose=1):
             self._scalar = False
 
         self._set_derivatives()
+        sub_obj_jac_chunk_sizes_are_ints = [
+            isposint(obj._jac_chunk_size) for obj in self.objectives
+        ]
+        warnif(
```

Comment:
I could see possibly making this an error instead of a warning? While technically the code can run with this inconsistency, it may indicate that the user meant to decrease memory usage but didn't use the right args, meaning it's probably better to fail early rather than waiting until OOM errors

Reply:
yea I think that is a good idea